src/Neo.CLI/CLI/MainService.cs

```diff
@@ -377,15 +378,46 @@ public async void Start(CommandLineOptions options)
             CustomApplicationSettings(options, Settings.Default);
             try
             {
-                NeoSystem = new NeoSystem(protocol, Settings.Default.Storage.Engine, string.Format(Settings.Default.Storage.Path, protocol.Network.ToString("X8")));
+                NeoSystem = new NeoSystem(protocol, Settings.Default.Storage.Engine,
+                    string.Format(Settings.Default.Storage.Path, protocol.Network.ToString("X8")));
             }
-            catch (DllNotFoundException)
+            catch (DllNotFoundException ex) when (ex.Message.Contains("libleveldb"))
             {
-                ConsoleHelper.Error("DLL not found, please install Microsoft Visual C++ Redistributable." + Environment.NewLine +
-                    "See https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist" + Environment.NewLine +
-                    "Press any key to exit.");
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    if (File.Exists("libleveldb.dll"))
+                    {
+                        ConsoleHelper.Error("Dependency DLL not found, please install Microsoft Visual C++ Redistributable." +
+                                            Environment.NewLine +
+                                            "See https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist" +
+                                            Environment.NewLine +
+                                            "Press any key to exit.");
+                    }
+                    else
+                    {
+                        ConsoleHelper.Error("DLL not found, please get libleveldb.dll." +
+                                            Environment.NewLine +
+                                            "Press any key to exit.");
+                    }
+
+                }
+                else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    ConsoleHelper.Error("DLL not found, please get libleveldb.dylib." +
+                                        Environment.NewLine +
+                                        "For AppleSilicon https://github.com/neo-project/neo/releases/download/v3.7.4/neo-cli-osx-arm64.zip" +
+                                        Environment.NewLine +
+                                        "For Intel https://github.com/neo-project/neo/releases/download/v3.7.4/neo-cli-osx-x64.zip" +
```

Comment:
Just send them to the release page. It much simpler to maintain.

Reply:
Agree